lab_4_retrieval_w_clustering/main.py

```diff
@@ -26,6 +28,10 @@ def get_paragraphs(text: str) -> list[str]:
     Returns:
         list[str]: Paragraphs from document.
     """
+    if not isinstance(text, str) or not text or text == '':
```

Comment:
`not text` and `text == ''` are the same

Reply:
done